### ISSUE TITLE
Libreoffice doesn't include alt text when it's set in the document from word

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble#34]
Tested with docassemble 1.9.8's current version of Libreoffice (24.2.7.2) and the latest available release (26.2.3), both fail. Work around is to manually set the alt text in libreoffice instead of using word. 

Should consider fixing upstream.

Related bugs / links:

* https://www.reddit.com/r/libreoffice/comments/13lswgf/losing_image_alt_texts_when_exporting_a_text/
* https://bugs.documentfoundation.org/show_bug.cgi?id=148952
* https://bugs.documentfoundation.org/show_bug.cgi?id=143311
* https://bugs.documentfoundation.org/show_bug.cgi?id=139007
